reference/configuration/doctrine.rst

```diff
@@ -499,4 +499,135 @@ can be placed directly under ``doctrine.orm`` config level.
+                    AppBundle:
+                        dir: Resources/config/doctrine
+                        
+    .. code-block:: xml
+    
+        <?xml version="1.0" charset="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:doctrine="http://symfony.com/schema/dic/doctrine">
+        
+            <doctrine:config>
+                <doctrine:orm auto-mapping="true">
+                    <mapping name="AppBundle" dir="Resources/config/doctrine" />
+                </doctrine:orm>
+            </doctrine:config>
+        </container>
+        
+    .. code-block:: php
+    
+        $container->loadFromExtension('doctrine', array(
+            'orm' => array(
+                'auto_mapping' => true,
+                'mappings' => array(
+                    'AppBundle' => array('dir' => 'Resources/config/doctrine'),
+                ),
+            ),
+        ));
+
+You can also create new mappings. For example outside of the Symfony folder.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+    
+        doctrine:
+                # ...
+                orm:
+                    # ...
+                    mappings:
+                        # ...
+                        SomeEntityNamespace:
+                            type: annotation
+                            dir: %kernel.root_dir%/../src/Entity
+                            is_bundle: false
+                            prefix: App\Entity
+                            alias: App
+                            
+    .. code-block:: xml
+    
+        <?xml version="1.0" charset="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:doctrine="http://symfony.com/schema/dic/doctrine">
+        
+            <doctrine:config>
+                <doctrine:orm>
+                    <mapping name="SomeEntityNamespace" 
+                        type="annotation"
+                        dir="%kernel.root_dir%/../src/Entity"
+                        is-bundle="false"
+                        prefix="App\Entity"
+                        alias="App"
+                    />
+                </doctrine:orm>
+            </doctrine:config>
+        </container>
+        
+    .. code-block:: php
+            
+        $container->loadFromExtension('doctrine', array(
+            'orm' => array(
+                'auto_mapping' => true,
+                'mappings' => array(
+                    'SomeEntityNamespace' => array(
+                        'type'      => 'annotation',
+                        'dir'       => '%kernel.root_dir%/../src/Entity',
+                        'is_bundle' => false,
+                        'prefix'    => 'App\Entity',
+                        'alias'     => 'App',
+                    ),
+                ),
+            ),
+        ));
+
+Detecting a Mapping MetadataDriver
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the ``type`` on the bundle configuration isn't set,
```

Comment:
what happens if `is_bundle` is false? Does it still look for the same paths? If so, we should remove the word "bundle" below, and replace them with "directory".